airbyte-integrations/connectors/source-my-hours/unit_tests/test_streams.py

```diff
@@ -0,0 +1,87 @@
+#
```

Comment:
nit: as you do not have a `streams.py` module these tests could go to `test_sources.py`. Or you could move your streams to a `streams.py` module.

Reply:
I moved the stream class to a separate stream.py file